.github/actions/should_run_monty/action.yml

```diff
@@ -15,20 +22,20 @@ runs:
   using: "composite"
   steps:
     - name: Filter by path for a pull request
-      if: ${{ github.event_name == 'pull_request' }}
+      if: ${{ inputs.event_name == 'pull_request' }}
```

Comment:
should this be inputs.git_event_name to follow the convention of the other github passthroughs?

Reply:
I don't think so. I started with `github_sha` and `github_base_ref`, but then renamed to `git_sha` and `git_base_ref` because `sha` and `base_ref` are `git` concepts (not GitHub). A pull request and an event name are GitHub concepts. I could rename it to `github_event_name`.

Reply:
Renamed to `github_event_name`.